pkgs/tools/inputmethods/skk/skk-dicts/default.nix

```
@@ -25,6 +25,8 @@ let
     url = "https://raw.githubusercontent.com/skk-dev/dict/8b35d07a7d2044d48b063d2774d9f9d00bb7cb48/SKK-JISYO.assoc";
     sha256 = "1smcbyv6srrhnpl7ic9nqds9nz3g2dgqngmhzkrdlwmvcpvakp1v";
   };
+
+  iconvBin = if stdenv.isDarwin then libiconv else  buildPackages.stdenv.cc.libc;
```

```suggestion
  iconvBin = if stdenv.buildPlatform.isDarwin then buildPackages.libiconv else buildPackages.stdenv.cc.libc;
```

same goes for the other package